assets/dev/js/frontend/handlers/container/handles-position.js

```diff
@@ -30,24 +26,18 @@ export default class HandlesPosition extends elementorModules.frontend.handlers.
 			return;
 		}
 
-		const isOverflowHidden = this.isOverflowHidden();
-
-		if ( ! isOverflowHidden && ! this.isFirstContainer() ) {
+		if ( ! this.isOverflowHidden() ) {
 			return;
 		}
 
-		const offset = isOverflowHidden ? 0 : this.getOffset(),
+		const offset = this.getOffset(),
 			$handlesElement = this.$element.find( '> .elementor-element-overlay > .elementor-editor-section-settings' ),
 			insideHandleClass = 'e-handles-inside';
 
 		if ( offset < 25 ) {
 			this.$element.addClass( insideHandleClass );
 
-			if ( offset < -5 ) {
-				$handlesElement.css( 'top', -offset );
-			} else {
-				$handlesElement.css( 'top', '' );
-			}
+			$handlesElement.css( 'top', offset < -5 ? -offset : '' );
```

Comment:
your choice, i prefer
```suggestion
                         const handlerOffset = offset < -5 ? -offset : '';
			$handlesElement.css( 'top', handlerOffset );
```

Reply:
I like it